packages/app/src/cli/models/app/loader.ts

```diff
@@ -497,7 +498,17 @@ class AppConfigurationLoader {
   async loaded() {
     const appDirectory = await this.getAppDirectory()
     let configSource: LinkedConfigurationSource = this.configName ? 'flag' : 'cached'
-    this.configName = this.configName ?? getCachedAppInfo(appDirectory)?.configFile
+
+    const cachedCurrentConfig = getCachedAppInfo(appDirectory)?.configFile
+    const cachedCurrentConfigPath = cachedCurrentConfig ? joinPath(appDirectory, cachedCurrentConfig) : null
+
+    if (!this.configName && cachedCurrentConfigPath && !fileExistsSync(cachedCurrentConfigPath)) {
+      const warningMessage = `Could not find config file ${cachedCurrentConfig}, please select a new config`
```

Comment:
When you have a single config file, it doesn't actually ask you to select (but it just uses it directly), so that `please select a new config` is not very accurate. What about removing it? It will show something similar in the next line if needed.

```suggestion
      const warningMessage = `Could not find config file ${cachedCurrentConfig}`
```

Reply:
Update based on @jenstanicak's suggestions 🙂